src/test/miner_tests.cpp

```diff
@@ -17,6 +17,7 @@
 #include "uint256.h"
 #include "util.h"
 #include "utilstrencodings.h"
+#include <boost/algorithm/string.hpp>
```

Comment:
Move down near other boost includes (try to keep it sorted too).

Reply:
I got rid of this dependency.